_posts/2019-02-11-lecture-08.md

```diff
@@ -0,0 +1,174 @@
+# Lecture 8 : Causal Discovery and Inference
+Causation implies correlation(or dependence) but correlation doesn't imply causation. Consider the case of chocolate consumption in a country compared with the number of nobel laureates in the country
+
+<figure>
+  <div class="row">
+    <div class="col two">
+      <img src="{{ '/assets/img/notes/lecture-08/scribe1.jpg' | relative_url }}"  width="600" height="400" />
+    </div>
+  </div>
+  <figcaption>
+    <strong>Chocolate consumptions vs Nobel Laureate</strong>
+  </figcaption>
+</figure>
+
+From this plot, we can see that they are highly correlated. But it is absurd to say that if we want to increase the number of nobel laureates in a country, we need to eat more chocolate. This is where the idea of causality comes in. They are correlated but chocolate consumption is not the cause of nobel laureates. The following equations capture this difference mathematically
```

Comment:
Grammar: Please change "nobel laureate" to "Nobel Laureate" everywhere in this file.